src/category_theory/limits/shapes/equalizers.lean

```diff
@@ -338,7 +340,21 @@ limit.hom_ext $ cone_parallel_pair_ext _ h
 
 /-- An equalizer morphism is a monomorphism -/
 instance equalizer.ι_mono : mono (equalizer.ι f g) :=
-{ right_cancellation := λ Z h k w, equalizer.hom_ext _ _ w }
+{ right_cancellation := λ Z h k w, equalizer.hom_ext w }
+
+end
+
+section
+variables {f g}
+/-- The equaliser morphism in any limit cone is a monomorphism. -/
+def mono_of_is_limit_parallel_pair {c : cone (parallel_pair f g)} (i : is_limit c) : mono (c.π.app zero) :=
+{ right_cancellation := λ Z h k w,
+  begin
+    apply i.hom_ext,
```

Comment:
If you are interested in a shorter proof:
```lean
 begin
    apply i.hom_ext,
    apply cone_parallel_pair_ext,
    exact w,
  end
```
or the term-mode version `i.hom_ext $ cone_parallel_pair_ext _ w`.

Reply:
Fantastic, thanks.